week2/homework/src/index.js

```diff
@@ -1,3 +1,82 @@
 'use strict';
 
 // TODO: Write the homework code in this file
+
+const { readFile, writeFile } = require('fs');
+const { promisify } = require('util');
+const readFileWithPromise = promisify(readFile);
+const writeFileWithPromise = promisify(writeFile);
+
+const args = process.argv.slice(2);
+
+const cmd = args[0];
+
+const TODO_PATH = 'todo.json';
+
+function readTodos() {
+    return readFileWithPromise(TODO_PATH, 'utf8')
+        .then(JSON.parse)
+        .catch(() => ([]));
+}
+
+function writeTodos(todos) {
+    return writeFileWithPromise(TODO_PATH, JSON.stringify(todos, null, 2));
+}
+
+function myHelper() {
+    console.log('Get help from these commands:1- node.add \'New todo\'   = Add a todo 2- node . delete [Delete item]  = Delete item from the list 3- node . list  = List all items 4- node . update [Item to be updated]  \'New todo\'  = Updating a todo. 5- node . reset  = Reset the list');
```

Comment:
You can split a string into multiple lines by using `\n` escape code like this `'line one\nline two\nand so on'`. Alternatively you can use [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) like this:

```js
const multiLineString = `line one
line two`;